package.json

```diff
@@ -34,78 +34,83 @@
     "README.md",
     "LICENSE"
   ],
+  "resolutions": {
+    "chokidar": "^3.5.1"
+  },
   "dependencies": {
-    "@babel/runtime": "7.13.10",
-    "bcryptjs": "2.3.0",
-    "body-parser": "1.19.0",
-    "codemirror-graphql": "github:timsuchanek/codemirror-graphql#details-fix",
-    "commander": "6.2.1",
-    "connect-flash": "0.1.1",
-    "cookie-session": "2.0.0-beta.3",
-    "copy-to-clipboard": "3.2.0",
-    "create-react-class": "15.7.0",
-    "csurf": "1.11.0",
-    "express": "4.17.1",
-    "graphql": "14.6.0",
-    "graphql-playground-react": "1.7.20",
-    "history": "4.10.1",
-    "immutable": "4.0.0-rc.9",
-    "immutable-devtools": "0.1.5",
-    "js-beautify": "1.11.0",
-    "json-file-plus": "3.2.0",
-    "package-json": "6.5.0",
-    "parse": "2.19.0",
-    "passport": "0.4.1",
-    "passport-local": "1.0.0",
-    "prismjs": "1.23.0",
-    "prop-types": "15.7.2",
-    "query-string": "6.14.1",
-    "react": "16.14.0",
-    "react-ace": "9.3.0",
-    "react-dnd": "10.0.2",
-    "react-dnd-html5-backend": "10.0.2",
-    "react-dom": "16.14.0",
-    "react-helmet": "6.0.0",
-    "react-json-view": "1.21.3",
-    "react-redux": "5.1.2",
-    "react-router": "5.1.2",
-    "react-router-dom": "5.1.2",
-    "regenerator-runtime": "0.13.5",
-    "semver": "7.3.4"
+    "@babel/runtime": "^7.13.10",
```

Comment:
Can you please remove all the range operators such as `^` in package.json?

Reply:
Why? It's package-lock.json for recording the exact versions.

Reply:
Because there are automated dependency checks that run on `package`, not `package-lock`.